packages/slider/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "prop-types": "^15.5.4",
+    "prop-types": "^15.7.2",
```

Comment:
(n-b) This is latest, I guess nothing in React 16+ changes proptype defs?

Reply:
shouldn't, and every other package was using 15.7.2 anyway - so this just resolves a warning. it was getting this version hoisted anyway